.goreleaser.yaml

```diff
@@ -163,6 +173,24 @@ dockers:
   - --label=org.opencontainers.image.version={{.Version}}
   - --label=org.opencontainers.image.source={{.GitURL}}
   use: buildx
+- goos: linux
+  goarch: arm
+  dockerfile: distributions/otelcol/Dockerfile
+  image_templates:
+  - otel/opentelemetry-collector:{{ .Version }}-arm
+  - ghcr.io/open-telemetry/opentelemetry-collector-releases/opentelemetry-collector:{{
+    .Version }}-arm
+  extra_files:
+  - configs/otelcol.yaml
+  build_flag_templates:
+  - --pull
+  - --platform=linux/arm
```

Comment:
This should have `arm64` or `arm32`, I don't believe `arm` by itself is a valid architecture.